pkg/ibmvpc/examples/out/explain_out/LBToIksNode_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,360 @@
+Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca to 192.168.4.4 (iks-node[192.168.4.4]) within ky-test-vpc
+========================================================================================================================================
```

Comment:
The user queried the path between two single addresses and got 9 (!) different paths.
This is unique to the LB.
Perhaps the incentive here  should be explained? add the explanation right after the header?

Reply:
Also, I did not verify the correctness of the output w.r.t. the input files.

Reply:
#674